docs/guides/docker.md

```diff
@@ -0,0 +1,256 @@
+---
+label: Docker automation
+order: -2
+description: Learn how to deploy a Docker stack for common media automation software.
```

Comment:
we don't use periods in headings/titles/etc. This is embed heading so remove it

Reply:
removed.